communication/slack-config/channels.yaml

```diff
@@ -25,6 +25,18 @@ channels:
   - name: cka-exam-prep
   - name: ckad-exam-prep
   - name: client-go-docs
+  - name: cloud-provider-aws
+    id: C0LRMHZ1T
+  - name: cloud-provider-azure
+    id: C5HJXTT9Q
+  - name: cloud-provider-gcp
+    id: C6WB33KNJ
+  - name: cloud-provider-ibmcloud
+    id: C90BWDWCE
+  - name: cloud-provider-openstack
+    id: C0LSA3T7C
+  - name: cloud-provider-vmware
```

Comment:
I have mixed feelings about this rename, this could either become `ug-vmware` (if the transition to user group happens contextually) or `cloud-provider-vsphere`.

Reply:
Good point, will rename to `cloud-provider-vsphere`. I think in the end there should be both channels, one for subproject development and the other for users.